### PR TITLE
Divider component

### PR DESCRIPTION
**Pull request by lhbrennan**
#### Description

Adds new `Divider` component

image

#### Scope
Minor: New Feature

### CODE REVIEW

File: src/divider/styled-components.ts [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  const borderResetStyles = {
+    borderRight: 'none',
+    borderBottom: 'none',
+    borderLeft: 'none',
+  };
+
+  if ($size === SIZE.cell) {
+    return {
+      ...borderResetStyles,
+      borderTop: `1px solid ${$theme.colors.borderOpaque}`,
```
Comment (tajo) [inline src/divider/styled-components.ts:22]: We should still probably keep using long hand props only to be safe?

**Review by tajo — APPROVED** [review]

File: src/divider/styled-components.ts (hunk 1 above)
Comment (lhbrennan) [reply src/divider/styled-components.ts]: Good catch

File: src/divider/styled-components.ts [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+Copyright (c) Uber Technologies, Inc.
+
+This source code is licensed under the MIT license found in the
+LICENSE file in the root directory of this source tree.
+*/
+import { styled } from '../styles/index';
+import { SIZE } from './constants';
+import type { DividerPropsT } from './types';
+
+export const StyledDivider = styled<'hr', DividerPropsT>('hr', (props) => {
```
Comment (chasestarr) [inline src/divider/styled-components.ts:11]:
```suggestion
import type { DividerProps } from './types';

export const StyledDivider = styled<'hr', DividerProps>('hr', (props) => {
```

File: src/divider/types.js.flow [hunk 3]
```diff
@@ -0,0 +1,13 @@
+/*
+Copyright (c) Uber Technologies, Inc.
+
+This source code is licensed under the MIT license found in the
+LICENSE file in the root directory of this source tree.
+*/
+// @flow
+import { SIZE } from './constants.js';
+
+export type DividerPropsT = {
```
Comment (chasestarr) [inline src/divider/types.js.flow:10]:
```suggestion
export type DividerProps = {
```

**Pull request merged**